.github/workflows/share-artifacts.yaml

```
@@ -0,0 +1,37 @@
+---
+
+on:
+  push: {}
+  workflow_dispatch: {}
+
+name: share artifacts
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    outputs:
+      artifacts-name: ${{ steps.artifacts.outputs.name }}
+      artifacts-link: ${{ steps.artifacts.outputs.link }}
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v3
+    - id: artifacts
```

Sure there is a "`-`" in front of "`id`" here ?

---

100% sure, but it looks weird because I forgot to put a name in the step. Might look better if I do this:
```suggestion
    - name: artifacts upload
      id: artifacts
```